plugins/catalog/api-report.md

```diff
@@ -29,17 +29,32 @@ import { UserListFilterKind } from '@backstage/plugin-catalog-react';
 
 // @public
 export interface AboutCardProps {
+  // (undocumented)
+  children?: React_2.ReactNode;
   // (undocumented)
   variant?: InfoCardVariants;
 }
 
 // @public (undocumented)
-export function AboutContent(props: AboutContentProps): JSX.Element;
```

Comment:
Is there a way we can keep backwards compatibility here? So that we don't break this component for people already using it?

Ideally we want to keep the `AboutContentProps` the same if we can, and just add the new `children` prop.

Reply:
Yes we can keep the entity here, but it won't be passed down to children since the children use `useEntity` hook to access the entity from the context.

It's possible to utilize the entity and pass it to children as props, but then the usage of those fields will be cluttered like `<MyCustomAboutField1 entity={entity} />`, so I took the suggestion https://github.com/backstage/backstage/issues/6175#issue-927956190 from @freben and unified them to use the `useEntity` hook. 

Pls let me know if you have a better solution so I can improve on this. If you are ok with this then I will proceed and just add the entity back to `AboutContentProps` for backward compatibility.

Reply:
So I think what we want to do, is put it back, but deprecate that prop, and `logger.warn` when it is being used.

Just so that we don't break everything on the next release of this plugin if people are using this.

You can deprecate using `tsdoc` by adding `@deprecated` annotation to the prop in the type and a nice description saying that it will only be allowed to be used in inside an `EntityProvider` in the future and the prop will be removed.